AsyncDisplayKit/ASMapNode.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+#import <MapKit/MapKit.h>
+@interface ASMapNode : ASControlNode
+- (instancetype)initWithCoordinate:(CLLocationCoordinate2D)coordinate NS_DESIGNATED_INITIALIZER;
+/**
+ This is the snapshot shot image node, this will be hidden (but not nil) when .liveMap = YES
+ */
+@property (nonatomic, readonly) ASImageNode *mapImage;
```

Comment:
@appleguy Yeah I've always wrestled with exposing this property, the main use case why I wanted to expose this property was allow developers to manipulate the image if they needed. Subclassing ASImageNode directly would solve both these problems so I'm in favour of going that route :)